src/utils.ts

```diff
@@ -1,33 +1,5 @@
 import { QueryObjType, GraphQLResolveInfo } from './types.ts';
 
-// const queryParser = (queryString: string): string => {
```

Comment:
Glad to see this ugly algo process gone in favor of such a better solution with the AST's.

Reply:
I am glad to see that go too. AST approach has a faster time complexity!